### PR TITLE
BI-969 - Blueberry Penetrometer Import

### PR DESCRIPTION
**Pull request by nickpalladino**
- Add additionalInfo support for Observations
- Add support for multiple observations per row
- Add Observation unit position / levels support
- Add Experimental Design PUI to study
- Some importer mapping bug fixes

### CODE REVIEW

File: src/main/java/org/breedinginsight/brapps/importer/model/base/ObservationUnit.java [hunk 1]
```diff
@@ -50,10 +52,16 @@
...
     @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
     @ImportFieldMetadata(id="observationLevel", name="Observation Level",
-            description = "The type of the observation unit. Example: Plot, Plant.")
+            description = "The type of the observation unit. Allowable options: plot, plant.")
     @ImportMappingRequired
     private String observationLevel;
 
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="observationLevelCode", name="Observation Level Code",
+            description = "An ID code for this level tag. Example: plot number for a plot observation level.")
```
Comment (ctucker3) [inline src/main/java/org/breedinginsight/brapps/importer/model/base/ObservationUnit.java:61]: Maybe something saying that plot numbers must be unique within a study?

**Review by ctucker3 — APPROVED** [review]
One small text suggestion, but looks good other than that.

**Review by dmeidlin — APPROVED** [review]

File: src/main/java/org/breedinginsight/brapps/importer/model/base/ObservationUnit.java (hunk 1 above)
Comment (nickpalladino) [reply src/main/java/org/breedinginsight/brapps/importer/model/base/ObservationUnit.java:61]: ya, I'll add that

Comment (nickpalladino) [reply src/main/java/org/breedinginsight/brapps/importer/model/base/ObservationUnit.java]: pushed changes

**Pull request merged**